grammars/css.cson

```diff
@@ -494,17 +494,15 @@
         ]
       }
       {
-        'begin': '(:)\\s*'
+        'begin': ':'
```

Comment:
What's the real difference? I mean, they're just spaces, does it matter if it's scoped as `property-list, property-value` or just `property-list`?